### DISCUSSION TITLE
图片和文献引用出现交错

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#137] [Q&A]
你好，
我发现这么一个问题。比如我们引用一篇文献在文中，文中的引用格式是（Li et al. 2022)。 但是这个位置是要断页的，于是变成 `（Li et`和新一页的`al.2022）`。但是这个时候，在新一页的顶部恰好自动放了一张图。于是顺序是：`（Li et`、新一页的顶部图、顶部图之下为：`al.2022）`。这个时候的图和`（Li et`、`al.2022）`具有相同的引用，也就是这个时候点击这个图片，会跳到文章末尾的（Li et al. 2022）文献。该怎么设置避免这个问题？
谢谢

**Comment by xkwxdyy** [comment]
手动换页，或者把图片的浮动体改为 `[b]`

**Reply by earthmoon** [reply]
你好，谢谢。改成`[b]`的确可以解决这个问题，但是有没有办法从根本上自动解决这个问题？就是参考文献因为换页，中间夹杂了图片，图片不会具有参考文献的超链接。

**Reply by xkwxdyy** [reply]
你最好能够提供代码还原这个问题，否则我们这不好测试。

**Reply by earthmoon** [reply]
木有问题，这个东西经常出现，下次出现我就发上来。

**Comment by sikouhjw** [comment]
论文是拿来看的，不是拿来点击的。

图片有超链接完全不影响『观看』。

**Reply by earthmoon** [reply]
> 论文是拿来看的，不是拿来点击的。
> 
> 图片有超链接完全不影响『观看』。

纸质版不影响，电子版不行啊，因为图片表格的caption或者tikz的图上的超链接全变了。
